lib/MetaCPAN/Script/Checksum.pm

```diff
@@ -0,0 +1,120 @@
+package MetaCPAN::Script::Checksum;
+
+use Moose;
```

Comment:
We could also use `MetaCPAN::Moose` here, I think.

Reply:
none of the scripts use it (yet), should be part of a separate change